src/js/services/conversation-service.js

```diff
@@ -89,7 +89,7 @@ export function sendMessage(text) {
 
         const {user} = store.getState();
 
-        return core().conversations.sendMessage(getUserId(), message).then((response) => {
+        return core().appUsers.sendMessage(getUserId(), message).then((response) => {
             if (!user.conversationStarted) {
```

Comment:
But that relies on a separate call to the `GET /messages` endpoint, right? So the store is in an inconsistent state until that time - the message is still considered not-uploaded until that call completes and the list is refreshed